### DISCUSSION TITLE
Option for scene view to not grab focus on mouse hover

### DISCUSSION THREAD

**Discussion by plyoung** [godotengine/godot-proposals#9918] [Editor]
Was not sure how to search for this so it might have been requested before, or perhaps there is such a feature?

When you move the mouse over the scene view, the 2D/3D view (not the "Scene" panel), it captures focus. The problem is, I might be busy in the FileSystem panel, selecting some file to delete, and my mouse moves over the scene view and it captures focus from the FileSystem panel and now when I hit delete I actually delete something from the scene!

An option to turn off this behaviour would be nice. I've done this accidentally several times now and I do not see the point to auto focus the view when it can grab focus when I click in it.

**Comment by Calinou** [comment]
> and I do not see the point to auto focus the view when it can grab focus when I click in it.

The reason this is done is to allow keyboard shortcuts to work immediately after focusing the 2D/3D viewport, without having to click within it (which could cause you to select another node, or lose your current selection).
